src/cascadia/TerminalApp/App.xaml

```diff
@@ -20,7 +20,8 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <!--  Include the MUX Controls resources  -->
-                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"
+                                       ControlsResourcesVersion="Version1" />

```

Comment:
Curious: Why did we have to add the version here?

Reply:
totally unfounded guess: this opts us out of the win11 styles, until we're ready to actually ingest them. Dustin did it in the original PR

Reply:
Correct.